home_work/hw_8/luhn_algo/main_luhn.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module is the main script to test the Luhn algorithm implementation.
+"""
+
+from home_work.hw_8.luhn_algo.cards_data import cards
+from home_work.hw_8.luhn_algo.luhn_algo import LuhnAlgorithm
+
+if __name__ == '__main__':
```

Comment:
you do not need to use this construction here:
https://builtin.com/articles/name-python#:~:text=Understanding%20If__Name__%3D%3D%20'__main__'%20in%20Python&text=This%20means%20that%20in%20such,will%20get%20executed%20once%20imported.